### PR TITLE
Recolour header elements

### PR DESCRIPTION
**Pull request by josebarata**
[#132239277] Recolour header elements

### CODE REVIEW

File: service/static/css-src/service.css [hunk 1]
```diff
@@ -60,14 +60,15 @@
 }
 
 .tab-nav .active a {
-    border: 1px solid #ffffff !important;
-    color: #5fb8ba !important;
+    border: 1px solid #fff !important;
+    background-color: #fff;
+    color: #518188;
```
Comment (tharglet) [inline service/static/css-src/service.css]: Active tab on blog is:
    background-color: #fff;
    color: #000;

**Review by tharglet — CHANGES_REQUESTED** [review]
Most colours are OK, one change please

**Comment by josebarata** [comment]
@tharglet Changes made!

**Review by tharglet — APPROVED** [review]

**Comment by tharglet** [comment]
happy with changes

**Pull request merged**